package.json

```diff
@@ -31,8 +31,8 @@
   },
   "scripts": {
     "coverage": "nyc --reporter=lcov npm test",
-    "test": "mocha -R spec",
-    "lint": "jshint lib/**.js test/**.js",
+    "test": "mocha -R spec --timeout 5000 test/index.js",
+    "lint": "jshint lib/**.js",
```

Comment:
Why not lint the tests anymore?  If you had an issue, can we just exclude that with a jshint ignore comment?

Reply:
Addressed in https://github.com/brightcove/hot-shots/pull/79/commits/34215ff69f34000493d3d897416f542837d65d1c.